javascript-package/tst/index.test.js

```diff
@@ -0,0 +1,399 @@
+import fetch from "isomorphic-fetch";
+import pluralize from "../src/pluralize";
+
+import Client from "../src";
+import { BASE_URL, ENTITY_TYPES, LEAGUES } from "../src/constants";
+
+import { playersXgoalsParameters } from "../src/parameters.js";
+
+// fetch payload mocks
```

Comment:
Thank you for using mocks here.

(We'll eventually go this route on the R/Python side. We just need to move all the existing tests over to the API repo. 😬)